.env.test

```diff
@@ -8,3 +8,5 @@ SITE_TITLE="Petition States Assembly (Test)"
 MEMCACHE_SERVERS=localhost:11211
 APPSIGNAL_APP_NAME=jpetitions-test
 GEOIP_DB_PATH=/path/to/GeoLite2-Country.mmdb
+PETITION_REPORT_EMAIL=gimmemyreport@example.com
```

Comment:
I would make this a site config value similar to `FEEDBACK_EMAIL` - the environment variable should drive the default value but should be overridable in the admin page for the site.

Reply:
Yes of course - much more flexible that way. Added (migration)[https://github.com/StatesOfJersey/e-petitions/pull/38/commits/1eb38a4384e8d48d3c4477d1fbdd1c35d8a26cb2#diff-4bb2a7b64380332de567b40284f18e4c] and (site default)[https://github.com/StatesOfJersey/e-petitions/pull/38/commits/1eb38a4384e8d48d3c4477d1fbdd1c35d8a26cb2#diff-4d2c8e068e0f67cc5b73443608489e4cR475].